packages/phone-number-privacy/combiner/src/common/input-validation.ts

```diff
@@ -18,7 +18,11 @@ export function isBodyReasonablySized(requestBody: any): boolean {
 }
 
 export function hasValidQueryPhoneNumberParam(requestBody: any): boolean {
-  return requestBody.blindedQueryPhoneNumber
+  return !!requestBody.blindedQueryPhoneNumber
+}
+
+export function hasValidPhoneNumberHash(requestBody: any): boolean {
```

Comment:
lots of anys here - possible to add some better typings in this file?

Reply:
Mostly not my code but okay I'll try to improve it

Reply:
These are currently shared across a few diff request types. We'd need to get creative with unions and generics to type them right. I'll get to it but rather not do it in this PR.